dist/images/ovnkube.sh

```diff
@@ -228,7 +249,7 @@ get_ovn_db_vars () {
 # This checks if OVS is up and running
 ovs_ready () {
   for daemon in `echo ovsdb-server ovs-vswitchd` ; do
-    pidfile=/var/run/openvswitch/${daemon}.pid
+    pidfile=${OVN_RUNDIR}/${daemon}.pid
```

Comment:
@numansiddique @dcbw this should be ${OVS_RUNDIR} since we are checking ovs-daemons here.

Reply:
Oops. Thanks for pointing this out.

Reply:
Done